packages/e2e/test/k6/scenarios/wallet-restoration.test.js

```diff
@@ -77,6 +77,15 @@ const hdWalletParams = {
   stakeAddrSearchGap: 5 * 2
 };
 
+/** OpenAPI versions. Needed to access provider endpoints */
+const apiVersion = {
```

Comment:
Ideally this would be updated when running `build:version` in `cardano-services-client`. Doesn't have to be in scope of this PR.

Reply:
Done here: a1d6daf863cb334adb0bcb334e4a4ba734543f9b.